### PR TITLE
add to cart + fix homepage

### PR DESCRIPTION
**Pull request by dungpt-0371**
## Related Tickets:
- [#68071](https://edu-redmine.sun-asterisk.vn/issues/68071)

## WHAT this PR does?
- BE xử lí thêm sách vào giỏ hàng
- Fix lại logic và giao diện trang homepage

## Notes Impacted Areas:
- app/Https/Controllers
- resources/views/layouts
- resources/views/books
- resources/views/welcome.blade.php
- routes/web.php

## Attachments:
image

### CODE REVIEW

**Comment by anhcnn-0378** [comment]
Oke

**Comment by dungpt-0371** [comment]
ready

File: app/Http/Controllers/CartController.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ public function store(Request $request)
 
             $cart->save();
         } else {
-            return redirect()->back()->withErrors(['error_cart' => 'The book already exists in the shopping cart']);
+            return redirect()->back()->withErrors(["error_cart_{$request['book']}"  => 'Already exists in your cart']);
```
Comment (ghost) [inline app/Http/Controllers/CartController.php]: localization?

**Comment by dungpt-0371** [comment]
ready

**Pull request merged**